misc/cluster/lda/corr-lda.stan

```diff
@@ -2,28 +2,26 @@ data {
   int<lower=2> K;               // num topics
   int<lower=2> V;               // num words
   int<lower=1> M;               // num docs
-  int<lower=1> N;               // total word instances
-  int<lower=1,upper=V> w[N];    // word n
-  int<lower=1,upper=M> doc[N];  // doc ID for word n
+  int<lower=0> corpus[M,V];     // word freq matrix, doc x word
   vector<lower=0>[V] beta;      // word prior
 }
 parameters {
   vector[K] mu;              // topic mean
   corr_matrix[K] Omega;      // correlation matrix
   vector<lower=0>[K] sigma;  // scales
-  vector[K] eta[M];           // logit topic dist for doc m
+  vector[K] eta[M];          // logit topic dist for doc m
   simplex[V] phi[K];         // word dist for topic k
 }
 transformed parameters {
   simplex[K] theta[M];       // simplex topic dist for doc m
   cov_matrix[K] Sigma;       // covariance matrix
   for (m in 1:M)
-    theta[m] <- softmax(eta[m]);
+    theta[m] = softmax(eta[m]);
   for (m in 1:K) {
-    Sigma[m,m] <- sigma[m] * sigma[m] * Omega[m,m];
```

Comment:
Thanks for replacing `<-`  --- that would be a good change for the original model, as well as replacing `increment_log_prob` with `target +=`.

`square(sigma[m])` will be a bit more efficient, as would be `sigma[m]^2`.